store/json.go

```diff
@@ -12,12 +12,16 @@ import (
 	"sync"
 	"time"
 
+	zapLog "github.com/Azure/azure-container-networking/cni/log"
```

Comment:
We should not need this as well since we are passing the logger during client initialization